src/Core/Components/NumberField/FluentNumberField.razor

```diff
@@ -24,7 +24,7 @@
                      max="@(ReadOnly ? value : Max)"
                      min="@(ReadOnly ? value : Min)"
                      id=@Id
-                     value="@value"
+                     current-value="@value"
```

Comment:
Are you sure that we need to use `current-value` ?
I can't find this attribute in [this documentation](https://github.com/microsoft/fluentui/blob/master/packages/web-components/src/number-field/number-field.vscode.definition.json).

Have you checked if using a **EditForm** with `mode=post` sends the value correctly?

Reply:
This is what I seein the FAST Explorer and on https://fluent-components.azurewebsites.net/?path=/docs/components-number-field--number-field:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/53d87e30-bb3e-4c07-ba9e-5475729ece7b)

Reply:
The web components in general , the value attribute is only used to set the initial value of a component.
- `<fluent-number-field value="5" />` renders the component with 5 as a starting value
- `<fluent-number-field current-value="7" />` renders the component with 7 as a starting value

So both work for initial value but only current-value works for (re)setting the component with a @bind-Value from the Blazor side.